testing/benchmark/Makefile

```diff
@@ -60,14 +55,21 @@ init:
 
 .PHONY: apply
 apply:
-	@terraform apply -var-file="es-system-profiles/$(ES_SYSTEM_PROFILE).tfvars" -auto-approve
+	@terraform apply -auto-approve
 
 .PHONY: destroy
 destroy:
 	@terraform destroy -auto-approve
 
+.PHONY: enable-benckmark-profile
+enable-benckmark-profile: 
```

Comment:
There's no need to declare this target here if we leave the Makefile vars as they were. A big drawback of having this dependency is the rigidity of the setup. I would rather have it extremely configurable at least for now and reevaluate if it becomes a problem.

The values of the variables can be overridden by exporting them in the corresponding Jenkins job before calling `run-benchmark`. I don't see a compelling reason to change the way this works for now.

Reply:
Now it just prints benchmark parameters